framework/src/base/AppFactory.C

```diff
@@ -51,8 +51,12 @@ AppFactory::createAppShared(const std::string & default_app_type,
   std::string app_type;
   if (!command_line->search("app_to_run", app_type))
     app_type = default_app_type;
+  else
+    mooseDeprecated("Please use [Application] block to specify application type, '--app <AppName>' "
+                    "will be deprecated in the future version.");
```

Comment:
```suggestion
                    "is deprecated and will be removed in a future release.");
```
it is now deprecated and it will be removed?

Reply:
Yes, Application/type does exactly the same thing